{{cookiecutter.project_slug}}/clients/web/vue3/src/composables/use-users.ts

```diff
@@ -14,15 +18,23 @@ import { reactive, ref } from 'vue'
 import { useRouter } from 'vue-router'
 import { useUserStore } from '@/stores/user'
 import { useAlert } from '@/composables/CommonAlerts'
+import { MustMatchValidator } from '@thinknimble/tn-forms'
 
 export function useUsers() {
   const userStore = useUserStore()
   const router = useRouter()
   const qc = useQueryClient()
-  const loginForm = reactive(new LoginForm({}))
-  const forgotPasswordForm = reactive(new EmailForgotPasswordForm({}))
-  const resetPasswordForm = reactive(new ResetPasswordForm({}))
-  const registerForm = reactive(new AccountForm({}))
```

Comment:
ahh need to fix this, we dont need to use the FormLevelValidator here and can just add it as it was